src/actions/index.js

```diff
@@ -0,0 +1,9 @@
+export const createBook = (book) => ({
+  type: 'CREATE_BOOK',
+  book
+})
+​
+export const removeBook = (book) => ({
```

Comment:
While the instructions (erroneously) indicated we should pass the whole `book` object into the `removeBook` action generator, the only thing we require is the `book`'s id. So let us change this action generator to just accept `id` as parameter (and thereby also including that `id` in the resulting action).